routes/web.php

```diff
@@ -19,6 +19,8 @@
   
   //user routes
   Route::get("/users/create/","UserController@create");
+  Route::get("auth/user","AuthenticateController@user");
+  Route::get("auth/refresh","AuthenticateController@user");
```

Comment:
Logically they might be different which is why there are two routes.  But in terms of actions they should perform the same thing.  Getting a user and refreshing the token.  When you get a user by the JWT convention is to invalidate the token and send a new one.  For refreshing a token all of our authentication routes except for the one invalidating the token return the user.  So this keep up with the consistency, and documentation also states this.